Wire up support for converting integrations-next configs to flow. (#5973)
grafana/agent: docs/sources/flow/getting-started/migrating-from-prometheus.md

After the configuration is converted, review the {{< param "PRODUCT_NAME" >}} configuration file created and verify that it's correct before starting to use it in a production environment.

> This following list of limitations is specific to the converter tooling and not {{< param "PRODUCT_NAME" >}}:

Reviewer: ```suggestion
The following list is specific to the convert command and not {{< param "PRODUCT_NAME" >}}:
```
